packs/feats/archetype/juggler/juggler-dedication.json

```diff
@@ -12,7 +12,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>You become trained in Performance; if you were already trained in Performance, you instead become trained in a skill of your choice. You also gain the @UUID[Compendium.pf2e.feats-srd.Item.Juggle] skill feat, even if you don't meet its prerequisites.</p>\n<p><strong>Special</strong> You cannot select another dedication feat until you have gained one other feat from the @UUID[Compendium.pf2e.journals.JournalEntry.vx5FGEG34AxI2dow.JournalEntryPage.bQeLZt29NnHEn6fh]{Juggler} archetype.</p>"
+            "value": "<p>You become trained in Performance; if you were already trained in Performance, you instead become trained in a skill of your choice. You also gain the @UUID[Compendium.pf2e.feats-srd.Item.Juggle] skill feat, even if you don't meet its prerequisites.</p>\n<p>@UUID[Compendium.pf2e.journals.JournalEntry.vx5FGEG34AxI2dow.JournalEntryPage.bQeLZt29NnHEn6fh]{Juggler}</p>"
```

Comment:
This one uniquely says "... until you have gained **one** other feat.." (emphasis mine). Should we keep it?